app/assets/javascripts/editor/editor.js.erb

```diff
@@ -339,7 +339,7 @@ var CodeOceanEditor = {
                     CodeOceanEditor.lastDeltaObject = null;
```

Comment:
I am currently not sure whether we should change the `lastDeltaObject` as well. In theory, I imagine that a race condition could occur: The `lastDeltaObject` is "global", in a sense that it is used for all open editors (i.e., all files). Potentially, there could be the same change in two distinct files. If, and that is very unlikely, the second change and event handler is faster than the assignment of `CodeOceanEditor.lastDeltaObject = null;`, we would ignore the second change (and, for example, not send it through the WebSocket connection).

Practically, I don't think that this could happen (due to human timing), but just wanted to point it out.